MainModule/Client/UI/Default/UserPanel.lua

```diff
@@ -296,12 +296,7 @@ return function(data, env)
 	end
 
 	if window then
-		local playerData   = Remote.Get("PlayerData")
-		local chatMod 	   = Remote.Get("Setting",{"Prefix","SpecialPrefix","BatchKey","AnyPrefix","DonorCommands","DonorCapes"})
-		local settingsData = Remote.Get("AllSettings")
-
-		Variables.Aliases = playerData.Aliases or {}
-
+		local commandPrefix = ":"
```

Comment:
woah woah woah whats this. what about when the prefix isn't `:`

Reply:
what happened to chatMod??????? GRRRRRRR

Reply:
who knows

Reply:
or you can just 
![image](https://github.com/Epix-Incorporated/Adonis/assets/54174182/54904460-5071-4523-b177-63b13de7d1ca)

Reply:
Exactly